docs/src/userguide/source_files_and_compilation.rst

```diff
@@ -146,6 +149,31 @@ documentation`_. To compile the extension for use in the current directory use:
 
     $ python setup.py build_ext --inplace
 
+.. note::
+
+    setuptools 74.1.0 adds experimental support of the extensions in :file:`pyproject.toml`:
```

Comment:
```suggestion
    setuptools 74.1.0 adds experimental support for extensions in :file:`pyproject.toml` (instead of :file:`setup.py`):
```

Just trying to make this clearer than this is replacing setup.py